### ISSUE TITLE
NTRs calcium terms

### ISSUE THREAD

**Issue by fypoadmin** [pombase/fypo#300]
I have one more figure to do on this paper so might add one or more requests tomorrow after our group meeting. I'll let you know.

1. normal immediate intracellular calcium spike following extracellular calcium stimulus (child of FYPO:0001199)
2. increased immediate intracellular calcium spike following extracellular calcium stimulus (child of FYPO:0001198?)

(OK to put it this way? -the resting Ca2+ resting state in pombe is 100-200 nM, when Ca2+ is administered extracellularly there is a very sharp spike in intracellular calcium concentration (immediate i.e. within 2 minutes of administration of stimulus) followed by decay) 

3.increased transcription from CRDE promoter (SO:0001865) in the absence of calcium / increased basal transcription from CDREpromoter 
4. absent down regulation/attenuation of transcription from CDRE promoter in response to calcium
(max transcription is not affected, but basal transcription is increased, and the response is not attenuated. See fig 3b)
(this was actually a reporter gene assay, where the reporter gene is downstream of several repeats of CDRE. I think it is fine to annotate to 'increased transcription' rather than 'increased protein level' because the protein in question doesn't have anything to do with pombe, we are only looking at transcription from a promoter involved in calcium signalling, in the absence of calcium). Not sure if this should be a child of FYPO:0000624 or FYPO:0000780, should these two terms be connected?

5. increased prolonged cellular calcium level  child of FYPO:0001198
6. normal prolonged cellular calcium level.
basically for these ones, I want to distinguish between the 'spike' response in requests 1 and 2, this is measured over 2 hrs instead of 10 minutes, fig 3d. What's annoying in the paper is that they don' show the data for WT cells, but in the text they compare it to wt, so I suppose they must have measured it. 

7. increased cellular calcium level in response to salt stress child of FYPO:0001198

8. resistant to tacrolimus (FK506)
9. normal cellular response to sorbitol and tacrolimus (FK506)
10. normal cellular response to salt stress and tacrolimus (FK506)
11. resistant to salt stress and tacrolimus (FK506)

12. normal cellular calcium level in response to salt stress 
13. normal nuclear morphology

Original comment by: Antonialock
